Assets/Scripts/Utility.cs

```diff
@@ -2,7 +2,22 @@
 using System.Collections.Generic;
 using UnityEngine;
 
-public class Utility : MonoBehaviour
+public class Utility
 {
-    
+    public static bool EqualSequence(List<string> a, List<string> b)
```

Comment:
Isnt this already included into standard library.

a.Equals(B) ?

Reply:
Unfortunatly not. Equals checks for the same reference, but I want to check for same Sequences. That is also possible but only with IEnumerable.SequenceEqual, not fors Lists...